### PR TITLE
G+, AP, YT integrations, APK Mirror site

### PR DESCRIPTION
**Pull request by Madis0**

### CODE REVIEW

**Comment by manastungare** [comment]
This is awesome, thanks @Madis0 !

Looks like the tests are failing, though. Could you please check?
https://travis-ci.org/chimbori/lite-apps/builds/176022394

You can also run `./gradlew check` locally before submitting.

File: lite-apps/Google+/manifest.json [hunk 1]
```diff
@@ -19,9 +19,17 @@
       "name": "__MSG_communities__",
       "url": "https://plus.google.com/communities/member"
     },
+	{
+      "name": "__MSG_profile__",
+      "url": "https://plus.google.com/u/0/me"
```
Comment (manastungare) [inline lite-apps/Google+/manifest.json]: Please remove the "/u/0", it’s only used for multi-login, and is automatically inserted if not already present.

File: lite-apps/APK Mirror/manifest.json [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  "hermit_bookmarks": [
+    {
+      "url": "http://www.apkmirror.com/developers/",
+      "name": "All Developers"
+    },
+    {
+      "url": "http://www.apkmirror.com/uploads/",
+      "name": "Latest Uploads"
+    },
+	{
```
Comment (manastungare) [inline lite-apps/APK Mirror/manifest.json]: Unindent by 2.

File: lite-apps/YouTube/manifest.json [hunk 3]
```diff
@@ -20,15 +20,50 @@
       "name": "Trending",
       "url": "https://www.youtube.com/feed/trending"
     },
+	{
```
Comment (manastungare) [inline lite-apps/YouTube/manifest.json]: Unindent by 3, here and everywhere.

File: lite-apps/YouTube/manifest.json [hunk 4]
```diff
@@ -20,15 +20,50 @@
...
       "url": "https://m.youtube.com/results?q=%s",
       "name": "Search"
     }
+  ],
+  "hermit_share": [
+    {
+      "url": "%u",
+      "name": "Open in lite app"
+    }
+  ],
```
Comment (manastungare) [inline lite-apps/YouTube/manifest.json]: Hermit will support sharing through other means. For now, please use Sharing for actions where text content is uploaded to the Lite App & processed directly.

**Comment by manastungare** [comment]
Please use square icons at 300x300.

**Comment by Madis0** [comment]
I think I fixed all problems now.

About indents and gradle - I didn't actually open the full project in Android Studio, just edited the files in Notepad++, that might be the cause of the errors.

**Comment by manastungare** [comment]
Thanks! I meant to unindent only the openings braces that were misaligned,
not the whole file! But don’t worry, I can take care of that.

**Comment by manastungare** [comment]
08:46:25.558 [DEBUG] [TestEventLogger] com.chimbori.liteapps.ValidatorTest > testAllManifestsAreValid FAILED
08:46:25.559 [DEBUG] [TestEventLogger]     java.lang.AssertionError: File [APK Mirror] is missing the field [manifest_url]

**Comment by Madis0** [comment]
Added manifest URL, added to lite-apps.json and tested everything with Android Studio (found out that I could've used gradle bat too but oh well). Hope everything's right now!

**Comment by manastungare** [comment]
Awesome, thanks for sending this in!

**Pull request merged**